tests/cases/compiler/globalFunctionAugmentationOverload.ts

```diff
@@ -0,0 +1,15 @@
+// @filename: mod.d.ts

```

Comment:
It's hard to tell what exactly this is testing -- could you try and simplify it a bit?